web/templates/web/_navigation.html

```diff
@@ -1,23 +1,70 @@
 {% load i18n %}
 {% load web_extras %}
 {% trans "Logout" as logout %}
-<header class="row">
-    <nav class="nav">
-        {% nav_link request 'web:home' 'Home' %}
-        {% if user.is_researcher %}
-            <a class="nav-link" href="{% url 'exp:study-list' %}">{% trans "Experimenter" %}</a>
-        {% endif %}
-        {% nav_link request 'web:studies-list' 'Studies' %}
-        {% nav_link request 'web:faq' 'FAQ' %}
-        {% nav_link request 'web:scientists' 'The Scientists' %}
-        {% nav_link request 'web:resources' 'Resources' %}
-        {% if user.is_authenticated %}
-            {% nav_link request 'accounts:manage-account' 'My Account' %}
-            {% nav_link request 'web:studies-history' 'My Past Studies' %}
-            {% nav_link request 'logout' logout %}
-        {% else %}
-            {% nav_signup request %}
-            {% nav_login request %}
-        {% endif %}
+<style>
+
+    ul.navbar-nav li.nav-item button.active,
+    ul.navbar-nav li.nav-item a.active {
+        --bs-nav-pills-link-active-bg: #e7e7e7;
+    }
+
+</style>
+<header class="row" id="navbar-row">
+    <nav class="navbar navbar-expand-md bg-light text-dark">
+        <div class="container-fluid">
+            <div class="me-auto order-0 w-100">
+                <ul class="navbar-nav nav-pills me-auto w-100">
+                    <li class="nav-item active text-center d-flex align-items-center">
+                        {% nav_link request 'web:home' 'Home' %}
+                    </li>
+                    {% if user.is_researcher %}
+                    <li class="nav-item text-center d-flex align-items-center">
+                        <a class="nav-link navbar-link link-secondary" href="{% url 'exp:study-list' %}">{% trans "Experimenter" %}</a>
+                    </li>
+                    {% endif %}
+                </ul>
+                <button class="navbar-toggler collapsed" type="button" data-bs-toggle="collapse" data-bs-target=".collapse1">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+            </div>
+            <div class="navbar-collapse collapse w-100 order-1 collapse1">
+                <ul class="navbar-nav nav-pills mx-auto justify-content-center w-100">
+                    <li class="nav-item text-center d-flex align-items-center">
+                        {% nav_link request 'web:studies-list' 'Studies' %}
+                    </li>
```

Comment:
Can we move the repetitive HTML into the `nav_link` template function?

Reply:
Yes!